franka_gazebo/src/franka_hw_sim.cpp

```diff
@@ -41,6 +41,10 @@ bool FrankaHWSim::initSim(const std::string& robot_namespace,
 
   ROS_INFO_STREAM_NAMED("franka_hw_sim", "Using physics type " << physics->GetType());
 
+  // Retrieve physics parameters
+  auto gravity_ = physics->World()->Gravity();
```

Comment:
This should be named simply `gravity` without the trailing underscore according to the [clang-tidy rules for variable naming](https://github.com/frankaemika/franka_ros/blob/develop/.clang-tidy#L21-L22).

Don't know why the CI didn't complain, will have a look into that

Reply:
Also please add a sentence in the CHANGELOG

Reply:
@gollth Done. I also rebased onto the `develop` branch.  Please let me know when I need to change something else. 

Please note that this pull request only reduces the drift but does not remove it.